CONTRIBUTING.md

```diff
@@ -134,6 +138,17 @@ Follow gitflow best practices as linked above.
 1. Merge `main` back into `develop`
 1. Push `develop`
 
-### DockerHub
+### Modifying Docker
+
+Follow this section when modifying the [Dockerfile](https://github.com/Sage-Bionetworks/Genie/blob/main/Dockerfile):
+
+1. Make sure you have your synapse config setup in your working directory
```

Comment:
What is your rationale of doing this over using env variable to log into the python client?

Reply:
No rationale actually, I think I just carried this over from another project (specifying the env variable would be easier). Let me modify the docker command so it only has relevant flags in the command.